src/schema/users.ts

```diff
@@ -169,12 +184,20 @@ export const typeDefs = /* GraphQL */ `
     """
     Get the reading rank of the user
     """
-    userReadingRank(id: ID!): ReadingRank
+    userReadingRank(id: ID!, version: Int): ReadingRank
     """
     Get the reading rank of the user
     """
     userMostReadTags(id: ID!): [MostReadTag]
     """
+    Get the reading top tags of the user
+    """
+    userReadingTopTags(
+      id: ID!
+      before: String!
+      after: String!
+    ): [TagsReadingStatus]
```

Comment:
What is this new query and what's the difference compared to `userMostReadTags`?

Reply:
This has the percentage - should we keep them in one query?

Reply:
Yes, please. Otherwise, it might be confusing